frontend/src/components/StatusTag.vue

```diff
@@ -131,60 +118,69 @@ export default {
     },
     chipTooltip () {
       return {
-        title: this.tag.name,
+        title: this.condition.name,
         status: this.chipStatus,
-        description: this.tag.description,
-        userErrorCodeObjects: filter(objectsFromErrorCodes(this.tag.codes), { userError: true })
+        description: this.condition.description,
+        userErrorCodeObjects: filter(objectsFromErrorCodes(this.condition.codes), { userError: true })
+      }
+    },
+    chipIcon () {
```

Comment:
The chip icon needs more margin. We could also use a `v-badge` for this use case. It will also safe some space.